tests/sqllogictests/suites/query/functions/02_0012_function_datetimes_tz.test

```diff
@@ -824,8 +824,10 @@ select to_datetime('1', '%s')
 ----
 1970-01-01 00:00:01.000000
 
-statement error 1006
+query T
 select to_timestamp('200,2000', '%s,%Y');
+----
+2000-01-01 00:03:20.000000
```

Comment:
Why the time part is 00:03:20?

And in this pr we modify the function "date_format" so the item of formatting maybe changed. So we need to know the new item and what's it means.

E.g. 
%Y -> means year